lib/ansible/modules/cloud/docker/docker_stack.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018 Dario Zanzico (git@dariozanzico.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: docker_stack
+author: "Dario Zanzico (@dariko)"
+short_description: docker stack module
+description:
+-   Manage docker stacks using the 'docker stack' command
+    on the target node
+    (see examples)
+version_added: "2.8"
+options:
+    name:
+        required: true
+        description:
+        -   Stack name
+    state:
+        description:
+        -   Service state.
+        default: "present"
+        choices:
+        -   present
+        -   absent
+    compose:
+        required: true
+        default: []
+        description:
+        -   List of compose definitions. Any element may be a string
+            referring to the path of the compose file on the target host
+            or the YAML contents of a compose file nested as dictionary.
+    prune:
+        required: false
+        default: false
+        description:
+        -   If true will add the `--prune` option to the `docker stack deploy` command.
```

Comment:
You cannot use backticks for formatting here. Use `C(--prune)` and `C(docker stack deploy)` instead.

Reply:
(There are some more places where you use backticks; I won't mark them all.)

Reply:
Thanks, I'm changing them